objection/commands/ios/hooking.py

```diff
@@ -319,3 +322,86 @@ def search_method(args: list) -> None:
 
     else:
         click.secho('No methods found')
+
+
+def enumerate(args: list) -> None:
+    """
+        Enumerates the current Android application for classes and methods.
```

Comment:
No it doesn't :P Rest of the function also has Android references.

Reply:
ah damn, to think this happened xD Thank you I'll fix this :)